src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: flex;
+  margin-right: 243px;
+}
+
+@media (min-width: 1024px) {
+  .header {
+    padding: 0 50px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .header-logo {
+    margin-right: 419px;
```

Comment:
Same as above, consider making this margin responsive, or using a different method to space elements, like flexbox or grid.